skyportal/models/localization.py

```diff
@@ -183,3 +204,61 @@ class LocalizationTile(Base):
     )
 
     healpix = sa.Column(healpix_alchemy.Tile, primary_key=True, index=True)
+
+
+class LocalizationProperty(Base):
+    """Store properties for localizations."""
+
+    update = delete = AccessibleIfUserMatches('sent_by')
+
+    sent_by_id = sa.Column(
+        sa.ForeignKey('users.id', ondelete='CASCADE'),
+        nullable=False,
+        index=True,
+        doc="The ID of the User who created this LocalizationProperty.",
+    )
+
+    sent_by = relationship(
+        "User",
+        foreign_keys=sent_by_id,
+        back_populates="localizationproperties",
+        doc="The user that saved this LocalizationProperty",
+    )
+
+    localization_id = sa.Column(
+        sa.ForeignKey('localizations.id', ondelete="CASCADE"),
+        primary_key=True,
+        index=True,
+        doc='localization ID',
+    )
+
+    data = sa.Column(JSONB, doc="Localization properties in JSON format.")
```

Comment:
If you don't index on this, that query for getting all the keys for all the data could be long.

Reply:
makes sense.